app/models/expression_file_info.rb

```diff
@@ -14,29 +14,43 @@ class ExpressionFileInfo
   UNITS_VALUES = [nil, 'UMI-corrected raw counts', 'raw counts']
   validates :units, inclusion: {in: UNITS_VALUES}
 
-  BIOSAMPLE_INPUT_TYPE_VALUES = [nil, 'whole cell', 'single nuclei', 'bulk']
+  BIOSAMPLE_INPUT_TYPE_VALUES = [nil, 'Whole cell', 'Single nuclei', 'Bulk']
   validates :biosample_input_type, inclusion: {in: BIOSAMPLE_INPUT_TYPE_VALUES}
 
   MULTIMODALITY_VALUES = [nil, 'CITE-seq', 'Patch-seq']
   validates :multimodality, inclusion: {in: MULTIMODALITY_VALUES}
 
   LIBRARY_CONSTRUCTION_VALUES = [nil,
-                                 'Smart-seq2/Fluidigm C1',
-                                 'MARS-seq',
-                                 'Seq-Well v1',
-                                 'Seq-Well S^3',
-                                 'inDrop',
-                                 'sci-RNA-seq',
                                  '10x 3\' v1',
                                  '10x 3\' v2',
                                  '10x 3\' v3',
                                  '10x 5\' v2',
                                  '10x 5\' v3',
                                  'CEL-seq2',
                                  'Drop-seq',
-                                 'SCRB-seq',
-                                 'ATAC-seq',
-                                 'ChIP-seq',
-                                 'methylomics']
+                                 'inDrop',
+                                 'Mars-seq',
+                                 'sci-RNA-seq',
+                                 'Seq-Well S^3',
+                                 'Seq-Well v1',
+                                 'Smart-like',
+                                 'Smart-seq2/Fluidigm C1',
+                                 # non-scRNAseq Assays
+                                 # scATACseq
+                                 'dsc-ATAC-seq',
+                                 'dsci-ATAC-seq',
+                                 'scATAC-seq/10x',
+                                 'scATAC-seq/Fluidigm',
+                                 'sci-ATAC-seq',
+                                 'scTHS-seq',
+                                 'snATAC-seq',
+                                 # spatial transcriptomics
+                                 '10x Visium',
+                                 'MERFISH',
+                                 'osmFISH',
+                                 'SeqFISH+',
+                                 'Slide-seq',
+                                 'smFISH',
+                                 'Drop-ChIP']
```

Comment:
For consistency with comments above to indicate assay types would add, as a comment (above Drop-ChIP):
single cell ChIP-seq assays